src/main/java/de/tum/in/www1/artemis/service/TextAssessmentService.java

```diff
@@ -17,12 +18,15 @@
 
     private final UserService userService;
 
+    private final ResultService resultService;
+
     public TextAssessmentService(UserService userService, ComplaintResponseService complaintResponseService, ComplaintRepository complaintRepository,
             FeedbackRepository feedbackRepository, ResultRepository resultRepository, TextSubmissionRepository textSubmissionRepository,
-            StudentParticipationRepository studentParticipationRepository, ResultService resultService, AuthorizationCheckService authCheckService) {
+            StudentParticipationRepository studentParticipationRepository, ResultService resultService, AuthorizationCheckService authCheckService, ResultService resultService) {
```

Comment:
I think you have reslutService twice here.

Reply:
I fixed it, thanks!